tests/unit/moduleapi/timer.tcl

```diff
@@ -54,7 +56,43 @@ start_server {tags {"modules"}} {
         assert_equal {} [r test.gettimer $id]
     }
 
-    test "Unload the module - timer" {
+    test "Module can be unloaded when timer was finished" {
+        r set "timer-incr-key" 0
+        r test.createtimer 2000 timer-incr-key
```

Comment:
Can this test case be faster too? 500 is probably enough even for slow machines.